server/handlers/policy_relationship_handler.go

```diff
@@ -0,0 +1,128 @@
+package handlers
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/layer5io/meshery/server/models"
+
+	"github.com/ghodss/yaml"
+	"github.com/open-policy-agent/opa/rego"
+	"github.com/sirupsen/logrus"
+)
+
+// Define a simple policy struct
+type Policy struct {
+	Rules string `json:"rules"`
+}
+
+type RelationObject struct {
+	DestinationID   string                 `json:"destination_id,omitempty"`
+	DestinationName string                 `json:"destination_name,omitempty"`
+	SourceId        string                 `json:"source_id,omitempty"`
+	SourceName      string                 `json:"source_name,omitempty"`
+	Port            map[string]interface{} `json:"port,omitempty"`
+}
+
+type NetworkPolicyRegoResponse struct {
+	ServicePodRelationships        []RelationObject `json:"service_pod_relationships,omitempty"`
+	ServiceDeploymentRelationships []RelationObject `json:"service_deployment_relationships,omitempty"`
+}
+
+// PolicyRelationshipRegoHandler gets the designFile in the bytes of Yaml and resolves the network relationship as of now.
+func PolicyRelationshipRegoHandler(ctx context.Context, designFile []byte) (*NetworkPolicyRegoResponse, error) {
+	// Load the policy
+	policyFile, err := ioutil.ReadFile("../meshmodel/policies/network-policy.rego") // absolute path, needs to be changed
+	if err != nil {
+		logrus.Fatal("error reading rego file", err.Error())
```

Comment:
Put these functions in Meshkit and wrap these errors in Meshkit errors.

Reply:
Done. https://github.com/meshery/meshkit/pull/311